app-modules/resource-hub/src/Filament/Actions/ChangeConcernStatusAction.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+/*
+<COPYRIGHT>
+
+    Copyright © 2016-2025, Canyon GBS LLC. All rights reserved.
+
+    Advising App™ is licensed under the Elastic License 2.0. For more details,
+    see https://github.com/canyongbs/advisingapp/blob/main/LICENSE.
+
+    Notice:
+
+    - You may not provide the software to third parties as a hosted or managed
+      service, where the service provides users with access to any substantial set of
+      the features or functionality of the software.
+    - You may not move, change, disable, or circumvent the license key functionality
+      in the software, and you may not remove or obscure any functionality in the
+      software that is protected by the license key.
+    - You may not alter, remove, or obscure any licensing, copyright, or other notices
+      of the licensor in the software. Any use of the licensor’s trademarks is subject
+      to applicable law.
+    - Canyon GBS LLC respects the intellectual property rights of others and expects the
+      same in return. Canyon GBS™ and Advising App™ are registered trademarks of
+      Canyon GBS LLC, and we are committed to enforcing and protecting our trademarks
+      vigorously.
+    - The software solution, including services, infrastructure, and code, is offered as a
+      Software as a Service (SaaS) by Canyon GBS LLC.
+    - Use of this software implies agreement to the license terms and conditions as stated
+      in the Elastic License 2.0.
+
+    For more information or inquiries please visit our website at
+    https://www.canyongbs.com or contact us via email at legal@canyongbs.com.
+
+</COPYRIGHT>
+*/
+
+namespace AdvisingApp\ResourceHub\Filament\Actions;
+
+use AdvisingApp\ResourceHub\Enums\ConcernStatus;
+use AdvisingApp\ResourceHub\Models\ResourceHubArticleConcern;
+use AdvisingApp\ResourceHub\Notifications\ResourceHubArticleConcernStatusChanged;
+use Filament\Actions\Action;
+use Filament\Forms\Components\Select;
+use Filament\Widgets\Widget;
+
+class ChangeConcernStatusAction extends Action
+{
+    protected function setUp(): void
+    {
+        parent::setUp();
+
+        $this
+            ->label('Change Status')
+            ->button()
+            ->outlined()
+            ->modalDescription('Select what status this concern should have.')
+            ->schema([
+                Select::make('status')
+                    ->options(ConcernStatus::class)
+                    ->enum(ConcernStatus::class)
+                    ->default(fn (ResourceHubArticleConcern $record) => $record->status->value),
+            ])
+            ->action(function (array $data, ResourceHubArticleConcern $record, Widget $livewire): void {
+                $record->status = $data['status'];
+
+                $record->save();
+
+                $record->createdBy->notifyNow(new ResourceHubArticleConcernStatusChanged($record));
```

Comment:
It may be better to centralize this by moving to to the `updated` of an observer.

That way also you can check in the Observer if the value was changed with model helpers and only dispatch this if it was actually changed.